common/lib/xmodule/xmodule/modulestore/tests/test_inheritance.py

```diff
@@ -0,0 +1,62 @@
+"""
+Unit tests for testing inheritance mixins
+"""
+from __future__ import absolute_import
+
+import ddt
+import unittest
+
+from mock import Mock
+from django.utils.timezone import now, timedelta
+from xblock.core import XBlock
+from xblock.fields import ScopeIds
+from xblock.runtime import DictKeyValueStore, KvsFieldData, Runtime
+from xblock.test.tools import TestRuntime
+
+from xmodule.modulestore.inheritance import InheritanceMixin
+
+
+class TestXBlock:
+    pass
```

Comment:
docstring

Reply:
Why this class is required `TestXBlock`?

Reply:
To create a new Xblock, containing the fields from InheritanceMixin